src/utils.h

```diff
@@ -203,6 +203,15 @@ typedef struct RelationSize
 	int64 index_size;
 } RelationSize;
 
+#define ADD_RELATIONSIZE(total, rel)                                                               \
+	do                                                                                             \
+	{                                                                                              \
+		total.heap_size += rel.heap_size;                                                          \
+		total.toast_size += rel.toast_size;                                                        \
+		total.index_size += rel.index_size;                                                        \
+		total_relsize.total_size += chunk_relsize.total_size;                                      \
+	} while (0)
```

Comment:
Why declare this macro here?? Do we expect use it outside of src/utils.c ?

Reply:
I'm always in favor of less is more, so if we're not using this macro in places that includes `src/utils.h` then make sense to make it in the scope of the `src/utils.c`. It's more or less we do in OOP making methods private or public (in this case it is in the private scope).

Reply:
Ok, moved.